application/dash_application/utility/df_manipulation.py

```diff
@@ -122,11 +122,11 @@ def get_pinery_samples_from_active_projects():
 
 def df_with_pinery_samples(df: DataFrame, pinery_samples: DataFrame, ius_cols:
                            List[str]):
-    """Do an outer merge between the DataFrame and modern Pinery samples
-    data."""
+    """Do an left merge between the DataFrame and modern Pinery samples
+    data. Only samples in DataFrame will be kept."""
```

Comment:
Can `Only samples in DataFrame will be kept.` be changed to `Only samples in left DataFrame will be kept.`?